### PR TITLE
media-video/obs-studio: Version bump to 18.0.2.

### PR DESCRIPTION
**Pull request by Chiitoo**
A simple version bump, and removal of some of the older versions.

While the upstream changes seem to be mostly interesting for non-Linux platforms, there are some bits for us, too: [18.0.1](https://github.com/jp9000/obs-studio/releases/tag/18.0.1)/[18.0.2](https://github.com/jp9000/obs-studio/releases/tag/18.0.2)

### CODE REVIEW

File: media-video/obs-studio/obs-studio-18.0.2.ebuild [hunk 1]
```diff
@@ -51,30 +51,32 @@ DEPEND="
...
-		-DOBS_MULTIARCH_SUFFIX="${libdir#lib}"
+		-DDISABLE_ALSA=$(usex !alsa)
+		-DDISABLE_FREETYPE=$(usex !truetype)
+		-DDISABLE_JACK=$(usex !jack)
+		-DDISABLE_LIBFDK=$(usex !fdk)
+		-DDISABLE_PULSEAUDIO=$(usex !pulseaudio)
+		-DDISABLE_V4L2=$(usex !v4l)
+		-DLIBOBS_PREFER_IMAGEMAGICK=$(usex imagemagick)
+		-DOBS_MULTIARCH_SUFFIX=${libdir#lib}
+		-DOBS_VERSION_OVERRIDE=${PV}
```
Comment (mgorny) [inline media-video/obs-studio/obs-studio-18.0.2.ebuild]: Why do we need to override version?

File: media-video/obs-studio/obs-studio-18.0.2.ebuild [hunk 2]
```diff
@@ -51,30 +51,32 @@ DEPEND="
...
 		-DUNIX_STRUCTURE=1
 	)
-
 	cmake-utils_src_configure
 }
 
+pkg_preinst() {
+	gnome2_icon_savelist
```
Comment (mgorny) [inline media-video/obs-studio/obs-studio-18.0.2.ebuild]: This is no longer necessary, due to the most recent gnome2-utils eclass changes.

File: media-video/obs-studio/obs-studio-18.0.2.ebuild (hunk 1 above)
Comment (Chiitoo) [inline media-video/obs-studio/obs-studio-18.0.2.ebuild]: Without the override, the non-git builds of the application show its version incorrectly as '0.0.1'.  I spotted this as the upstream solution back in #3270.

**Comment by mgorny** [comment]
Ok, thanks. I'll fix the icon call and merge it.

**Comment by Chiitoo** [comment]
Thanks!

**Pull request closed without merging**